src/pages/indicatorTypes/form/IndicatorTypesSchema.ts

```diff
@@ -0,0 +1,16 @@
+import { z } from 'zod'
+import { getDefaults, createFormValidate, modelFormSchemas } from '../../../lib'
+
+const { identifiable } = modelFormSchemas
+
+export const IndicatorSchema = identifiable.extend({
+    name: z.string().min(1, 'Name is required'),
+    factor: z
```

Comment:
I think the best way to handle `factor` being an integer when fetching, while also support string is to coerce it. Eg. 

```
factor: z.coerce.string().min(1, 'Factor is required')
```